setup.py

```diff
@@ -94,7 +94,7 @@
         ['templates/*', 'local_settings.py.example']},
       scripts=glob('bin/*'),
       data_files=webapp_content.items() + storage_dirs + conf_files + examples,
-      install_requires=['Django>=1.9,<1.9.99', 'django-tagging==0.4.3', 'pytz', 'pyparsing', 'cairocffi'],
+      install_requires=['Django>=1.9,<1.9.99', 'django-tagging==0.4.3', 'pytz', 'pyparsing', 'cairocffi', 'urllib3==1.19.1'],
```

Comment:
I agree, I've used https://docs.openstack.org/developer/pbr/ in the past with good success.  That's outside the scope of this PR though.